.gitignore

```diff
@@ -16,3 +16,4 @@ tsconfig.json
 
 **/package-lock.json
 package-lock.json
+./packages/storefront-lwc/scripts/api.mjs
```

Comment:
i dont think we need this here, because there is a gitignore within storefront-lwc that has this for api.js and api.mjs